CustomTabBar/TabButton.swift

```diff
@@ -40,6 +40,11 @@ class TabButton: UIView {
             updateInsets()
         }
     }
+    var titleFont: UIFont = UIFont.systemFontOfSize(16) {
+        didSet {
+            setupViews()
```

Comment:
This is a bit heavy for just changing the font. I think we should just have something simpler here:
`titleLabel.font = titleFont`